server/docker-compose.yml

```diff
@@ -7,7 +6,7 @@ services:
       dockerfile: Dockerfile
     restart: always
     ports:
-     - "80:80"
+    - "${FRONTEND_PORT:-80}:80"
```

Comment:
Is this variable defined somewhere else?

Reply:
In the sample environment file now, ya.